util/stmtsummary/statement_summary_test.go

```diff
@@ -75,11 +75,12 @@ func (s *testStmtSummarySuite) TestAddStatement(c *C) {
 		digest:     stmtExecInfo1.Digest,
 		planDigest: stmtExecInfo1.PlanDigest,
 	}
+	samplePlan, _ := stmtExecInfo1.PlanGenerator()
```

Comment:
Do we need to add test cases to detect the second return value `planHint` of the `PlanGenerator` function?

Reply:
The SQL tests in bindinfo package should have covered this testing purpose?

Reply:
OK. Please resolve the conflicts. Rest LGTM